File: tbg-random.java
import java.util.Random;

public class TheBadGuysJava {
  public static void main(String[] args) {
    String[] theBadGuys = {
      "Wolf",
      "Snake",
      "Tarantula",
      "Shark",
      "Piranha",
      "Diane Foxington"
    };

    Random r = new Random();
    String yourLove = theBadGuys[r.nextInt(theBadGuys.length)];

    if (yourLove = "Wolf" && yourLove = "Diane Foxington") {
      System.out.println("You have a crush on ").concat(yourLove).concat(" I see, you must be a furry!");
    } else {
      System.out.println("You chose ").concat(yourLove).concat(" as your crush uwu");
    }
  }
}

File: tbg-random.js
let theBadGuys = [
  "Mr. Wolf",
  "Mr. Snake",
  "Ms. Tarantula",
  "Mr. Piranha",
  "Mr. Shark",
  "Diane Foxington",
];

let yourLove = theBadGuys[Math.floor(Math.random() * theBadGuys.length)];

if (yourLove === "Diane Foxington" || yourLove === "Mr. Wolf") {
  console.log(`You have a crush on ${yourLove} you must be a furry!`);
} else {
  console.log(`You chose ${yourLove} as your crush uwu`);
}
